### PR TITLE
chore(6.2.6): 🎉

### PR DESCRIPTION
**Pull request by xHyroM**
- [x] channelThreadOnly (djs v13)
- [x] NUMBER Argument Type
- [x] More Colors
- [x] CommandInteraction
- [x] ButtonInteraction
- [x] SelectMenuInteraction
- [x] MessageComponentInteraction
- [x] usage (cmd option)
- [x] objectArgs for legacy cmds
- [x] `<MessageComponentInteraction>.clicker` deprecated
- [x] Multi guild prefix fix
- [x] Mentionable argument type for normal
- [x] Thread Channels support https://github.com/Garlic-Team/GCommands/pull/73
- [x] Stickers
- [x] Deafen typo 
https://github.com/Garlic-Team/GCommands/pull/83
- [x] fetchClientApplicaion 
https://github.com/Garlic-Team/GCommands/pull/87
- [x] Indonesian translation
https://github.com/Garlic-Team/GCommands/pull/86#pullrequestreview-724813797
- [x] @gcommands/events ^1.0.8
- [x] Support for more than 1 character in the prefix
- [x] fix inlineReply
- [x] `MessageComponent.id` > `MessageComponent.customId`
- [x] Italia translation
https://github.com/Garlic-Team/GCommands/pull/100
- [x] Command.reload
- [x] createMessageComponentCollector, awaitMessageComponents
- Deprecate awaitSelectMenus, awaitButtons, collectors 
- [x] Context Menus
- User Commands
- Message Commands
- [x] Events
- commandExecute 
- commandError
- [x] Optional arguments for normal (legal) commands
- [x] Rate limit resolve
obrázok

**GEvents**
- [x] `<Event>`.reload()

**Github**
- [x] Pull request template
- [x] label-approved
- [x] Publish dev workflow
- [x] ESLint
- [x] Guide 
https://github.com/Garlic-Team/GCommands/pull/76

### CODE REVIEW

**Review by Gabe616 — APPROVED** [review]
seems pog to me

File: src/structures/MessageButton.js [hunk 1]
```diff
@@ -1,94 +1,91 @@
...
+    gray: 2,
+    grey: 2,
+    green: 3,
+    red: 4,
+    url: 5,
+    primary: 1,
+    secondary: 2,
+    success: 3,
+    danger: 4,
+    link: 5,
```
Comment (SebestikCZ) [inline src/structures/MessageButton.js:17]: why is there comma at the end?

**Review by SebestikCZ — APPROVED** [review]
Approving, but Hyro, have on memory, that D.js v13 requires node v16.6+

https://github.com/discordjs/discord.js/pull/6390#issuecomment-896776705

**Comment by xHyroM** [comment]
> Approving, but Hyro, have on memory, that D.js v13 requires node v16.6+
> 
> [discordjs/discord.js#6390 (comment)](https://github.com/discordjs/discord.js/pull/6390#issuecomment-896776705)

I know

File: src/structures/MessageButton.js (hunk 1 above)
Comment (xHyroM) [reply src/structures/MessageButton.js:17]: Because eslint

**Comment by xHyroM** [comment]
Final testing...

**Pull request merged**